index.html

```diff
@@ -1,312 +1,383 @@
-              <li>
-                <div class="lang_sec">
-                  <img
-                    src="icons/rectangle.png"
-                    width="18px"
-                    height="18px"
-                    alt=""
-                  />
-                  <h2>Frameworks</h2>
-                </div>
-                <ul class="minor_menu">
-                  <li class="jss">Bootstrap</li>
-                  <li class="jss">Ruby on Rails</li>
-                  <li class="jss">RSPec</li>
-                  <li class="jss">CapyBara</li>
-                  <li class="jss">Selenium</li>
-                </ul>
-              </li>
-              <li>
-                <div class="lang_sec">
-                  <img
-                    src="icons/circle.png"
-                    width="18px"
-                    height="18px"
-                    alt=""
-                  />
-                  <h2>Skills</h2>
-                </div>
-                <ul class="minor_menu">
-                  <li class="jss">Codekit</li>
-                  <li class="jss">Github</li>
-                  <li class="jss">Codepen</li>
-                  <li class="jss">Gitlab</li>
-                  <li class="jss">Terminal</li>
-                </ul>
-              </li>
-            </ul>
+    <section class="contact">
+      <article class="contact_content">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="https://formspree.io/f/xgebdydo" method="post" class="contact_form">
+          <input name="FullName" type="text" placeholder="Full name" required maxlength="30" />
+          <input name="Email" type="email" placeholder="Email address" required />
+          <textarea name="Message" rows="4" cols="50" placeholder="Write me something..." maxlength="500" required></textarea>
+          <div class="centered_button">
+            <button type="submit">Get in touch</button>
           </div>
-        </article>
-		</section>
-    </main>
-  </body>
-</html>
+        </form>
+      </article>
+      <hr />
+      <div class="footer_icons">
+        <ul>
+          <li>
+            <a href="https://github.com/Kanyorok">
+              <span class="icon git_footer"></span>
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/robert-mbugua-a8745093/">
+              <span class="icon linked_footer"></span>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <span class="icon angel_footer"></span>
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/CarlosMbugua">
+              <span class="icon twitter_footer"></span>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <span class="icon medium_footer"></span>
+            </a>
+          </li>
+        </ul>
+      </div>
```

Comment:
- [OPTIONAL] Your design is awesome :clap: :clap: but there are  a  few things that could be improved to make it match the design more closely, and to make it a great design to pride about with friends :smile: 

_Check this:_

Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/235904892-0ae111b0-b9bc-4dcb-abba-774c5ba75afb.png)|![image](https://user-images.githubusercontent.com/99715640/235904947-26e0b6db-cd90-4e65-b0a2-b6fa7537889a.png)|
  - Looking at the figure above, one can almost notice a sharp contrast of:
    - [ ] [OPTIONAL] The missing `black divider` at the bottom base of the page, and
    - [ ] [OPTIONAL] The overflow indicated on the `x-axis` of your page when on a mobile screen of `375px` due to an element with some fixed width that's spanning beyond normal screen size. This is not really good for user experience as they will have to struggle with scrolling sideways before enjoying the beauty of interacting with your page :scream: 

_**I am confident that it will improve your design :muscle:**_